src/Worker/Cron.php

```diff
@@ -30,9 +30,6 @@ public static function execute($parameter = '', $generation = 0)
 		$last = Config::get('system', 'last_cron');
 
 		$poll_interval = intval(Config::get('system', 'cron_interval'));
-		if (! $poll_interval) {
```

Comment:
You can use `Config::get('system', 'cron_interval', 10);` instead

Reply:
We don't need this, since the default value in ```config/defaults.config.php``` is now defined. Because of this I removed that part. In fact the problem occurred, because we queried the config value at another place where we hadn't got any default value defined.